### PR TITLE
614 add product to cart - DONE.

### PR DESCRIPTION
**Pull request by TalKoren11**
Added quntity funcintlity to domain too.

### CODE REVIEW

File: src/main/java/Domain/Facades/ShoppingCartFacade.java [hunk 1]
```diff
@@ -94,10 +94,10 @@ public void addCartForUser(String guestID, User user) {
      * This method called when a user add a product to his cart.
      */
     @Transactional
-    public void addProductToUserCart(String userName, int productID, int shopID) throws StockMarketException {
+    public void addProductToUserCart(String userName, int productID, int shopID, int quantity) throws StockMarketException {
```
[the remark this answers is not in the document]
Comment (inbarbc) [reply src/main/java/Domain/Facades/ShoppingCartFacade.java:97]: think about:
* what happend to iligal argument of quantity?
* what if there is less products then quantity in shop?

**Review by inbarbc — APPROVED** [review]

**Comment by inbarbc** [comment]
pls vies and think about the comments

File: src/main/java/Domain/Facades/ShoppingCartFacade.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (TalKoren11) [reply src/main/java/Domain/Facades/ShoppingCartFacade.java:97]: the UI let you enter only positive numbers, but it'll be better to check it in domain too ty.
for the second question, the purchase card wont work

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -13,10 +13,10 @@ spring.jpa.defer-datasource-initialization = true
 jwk.secret=c2VjcmV0S2V5QmFzZTY0RW5jb2RlZA==
 
 # Database connection settings for SQL Server
-spring.datasource.url=jdbc:sqlserver://stock-market.database.windows.net:1433;database=Production;encrypt=true;trustServerCertificate=false;hostNameInCertificate=*.database.windows.net;loginTimeout=30;
-spring.datasource.username=stockmarket_admin@stock-market
-spring.datasource.password=Aa123456!
-spring.datasource.driver-class-name=com.microsoft.sqlserver.jdbc.SQLServerDriver
+# spring.datasource.url=jdbc:sqlserver://stock-market.database.windows.net:1433;database=Production;encrypt=true;trustServerCertificate=false;hostNameInCertificate=*.database.windows.net;loginTimeout=30;
```
[the remark this answers is not in the document]
Comment (TalKoren11) [reply src/main/resources/application.properties:16]: oops sorry the db didnt work for me

**Pull request merged**